src/promptflow/tests/sdk_cli_test/unittests/test_exceptions.py

```diff
@@ -20,15 +29,15 @@ def test_error_category_with_unknow_error(self, pf):
         except Exception as e:
             ex = e
         error_category, error_type, error_target, error_message, error_detail = _ErrorInfo.get_error_info(ex)
-        assert error_category == ErrorCategory.UNKNOWN
+        assert error_category == ErrorCategory.SYSTEM_ERROR
```

Comment:
As the test name says, it is aimed to test the error category with unknown errors. Please verify it breaks the original intent. If it is by design, please rename the test case name.

Reply:
you are right, I will rename the test name.